### DISCUSSION TITLE
this.config.disablePriceReload

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1935] [Questions & Answers]
I see this in the javascripts of magento, but where can you set these variables?

this.config.disablePriceReload

**Comment by luigifab** [comment]
I suggest to you to search "disablePriceReload" in all OpenMage files.
You will found: `app/code/core/Mage/Adminhtml/Block/Catalog/Product/Composite/Fieldset/Configurable.php:72:        $result['disablePriceReload'] = true; // There's no field for price at popup`

**Comment by kiatng** [comment]
You can also search like this: https://github.com/OpenMage/magento-lts/search?q=disablePriceReload

**Comment by f1-outsourcing** [comment]
Yes but it is still unclear to me where you can configure this variable. Can I set this somewhere in the admin web interface?
